onebusaway-android/src/amazon/java/org/onebusaway/android/map/googlemapsv2/VehicleOverlay.java

```diff
@@ -91,25 +92,30 @@
 
     private static final int NORTH = 0;
 
-    private static final int NORTH_WEST = 1;
```

Comment:
I tested the app and the icons are pointing the right direction. I looked over the diff and I found that the halfwinds do go clockwise. The constants were used in the former method createvehicleicon to retrieve the icon for a particular direction. I now use the constants to determine which index the icon should go, which was not the case before.